docker-for-mac/index.md

```diff
@@ -94,7 +94,7 @@ Choose ![whale menu](/docker-for-mac/images/whale-x.png){: .inline} -> **Prefere
 
 ![Docker context menu](images/menu/d4m-menu-prefs.png){:width="250px"}
 
-### General tab
+### General
```

Comment:
editing for style? ;)

Reply:
Yep.  As written in the commit message, there are many gratuitous differences between both docs, and I tried to harmonize them, choosing what seemed most appropriate.  And in many occurrences, the shortest was the best (IMHO).